### ISSUE TITLE
Read from $PATH for commands to run

### ISSUE THREAD

**Issue by ReneFroger** [bmwant/hapless#4]
I have the [tye](https://github.com/dotnet/tye), an executable. `type tye` gives me this:  `/home/user/.dotnet/tools/tye`.

Moving this to `/usr/bin/  will not work.  Tye which needs dependencies which are stored in dotnet/tools. But when I do `hap run tye`, I get the error: `/bin/sh: 1: tye not found`.

But dotnet/tools is in my PATH. So I think Hapless should include the local path as well when running commands?

**Comment by bmwant** [comment]
Right, agree that it should automatically pickup whatever is possible to run from current shell session. I will look into that and fix this issue.

**Comment by BionicRust** [comment]
just reboot
